noir/noir-repo/test_programs/execution_success/verify_honk_proof/src/main.nr

```diff
@@ -6,7 +6,7 @@ fn main(
     // This is the proof without public inputs attached.
     // 
     // This means: the size of this does not change with the number of public inputs.
-    proof: [Field; 153],
+    proof: [Field; 156],
```

Comment:
+3 field elements in proof is due to loss of one evaluation (-1) and the addition of 1 commitment (+4)